backend/secrets.py

```diff
@@ -2,3 +2,6 @@
 # https://console.firebase.google.com/project/trogdors-29fa4/settings/serviceaccounts/databasesecrets
 FIREBASE_SECRET = 'd0h365xNGuJU7A3rB65bAJ4v59VMLJiaNY5R0D6C'
 FIREBASE_EMAIL = ''
+IONIC_TOKEN = ''
+IONIC_APPID = ''
+IONIC_SECURITY_TAG = ''
```

Comment:
Might be worth having a simple shell script that does the deploy.
First time it's ran, we can prompt for the secrets and auto-create a secrets.py in some ~/.config path then on deploy, run the pip command, copy the file in and run the gcloud deploy.